### PR TITLE
[story/ECP-1199] Add qualification name to admin ECP claim qualification task presenter

### PR DESCRIPTION
**Pull request by nomasprime**

### CODE REVIEW

File: app/models/early_career_payments/admin_tasks_presenter.rb [hunk 1]
```diff
@@ -26,7 +26,14 @@ def identity_confirmation
     def qualifications
       [].tap do |a|
         a << [
-          "ITT start/end year",
+          "Qualification",
+          I18n.t("early_career_payments.answers.qualification.#{eligibility.qualification}")
+        ]
+
+        year_type = eligibility.postgraduate_itt? ? "start" : "end"
```
Comment (WorkSutton) [inline app/models/early_career_payments/admin_tasks_presenter.rb:33]: @nomasprime from our enum on qualification correct?

**Review by WorkSutton — COMMENTED** [review]
@nomasprime Couple of questions for clarification

File: spec/models/automated_checks/claim_verifiers/qualifications_spec.rb [hunk 2]
```diff
@@ -20,15 +20,15 @@ module ClaimVerifiers
...
                   "niNumber": "#{data[:national_insurance_number]}",
-                  "qtsAwardDate": "#{data[:qts_award_date] || DateTime.now}",
+                  "qtsAwardDate": "#{data[:qts_award_date] || "2015-09-01T00:00:00+00:00"}",
                   "ittSubject1Code": "#{data.dig(:itt_subject_codes, 0)}",
                   "ittSubject2Code": "#{data.dig(:itt_subject_codes, 1)}",
                   "ittSubject3Code": "#{data.dig(:itt_subject_codes, 2)}",
                   "activeAlert": true,
                   "qualificationName": "#{data[:qualification_name] || "BA"}",
-                  "ittStartDate": "#{data[:itt_start_date] || DateTime.now}"
+                  "ittStartDate": "#{data[:itt_start_date] || "2015-09-01T00:00:00+00:00"}"
```
[the remark this answers is not in the document]
Comment (nomasprime) [reply spec/models/automated_checks/claim_verifiers/qualifications_spec.rb:31]: Nope, arbritrary value.

File: app/models/early_career_payments/admin_tasks_presenter.rb (hunk 1 above)
Comment (nomasprime) [reply app/models/early_career_payments/admin_tasks_presenter.rb:33]: Correct.

File: spec/models/automated_checks/claim_verifiers/qualifications_spec.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (WorkSutton) [reply spec/models/automated_checks/claim_verifiers/qualifications_spec.rb:31]: Oh ok, any reason for so long ago? Is it to roughly marry up with timelines?

**Review by WorkSutton — APPROVED** [review]

File: spec/models/automated_checks/claim_verifiers/qualifications_spec.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (nomasprime) [reply spec/models/automated_checks/claim_verifiers/qualifications_spec.rb:31]: Nope, completely arbitrary.

**Pull request merged**